### DISCUSSION TITLE
最新版本开启绕过指定区域 IP以后谷歌商店无法更新应用

### DISCUSSION THREAD

**Discussion by daoyelihuowang** [vernesong/OpenClash#4122] [Q&A]
一直转圈圈等待中本人小白不知道什么情况请各位大佬解答，绕过指定区域 IPv4 黑名单默认没有动过

**Comment by fqscfqj** [comment]
绕过指定区域黑名单添加这几个
services.googleapis.cn
googleapis.cn
xn--ngstr-lra8j.com
gstatic.com
